### ISSUE TITLE
Inconsistency with reference to form uid in answers

### ISSUE THREAD

**Issue by sypets** [in2code-de/powermail#937]
It is unclear to me: Can a form be used in several plugins? This is not prevented, but doing so results in some problems:

* the form uid is stored in the answers `tx_powermail_domain_model_mail.form`, so there is a reference to the form, not to the powermail plugin or page where the form is used
* in the plugin, some answers are shown. Which answers are shown, depends on the uid of the form, not the plugin, so answers may get shown which belong to a different plugin

Example:

* form "Register"  uid=333
  * used in
    *  plugin "Register for event: Hybrid work"
    * plugin "Register for even: The end of capitalism"

Now, the incoming answers will all contain the reference to form 33, but on the plugin of "End of capitalism", the answers for "Hybrid work" are displayed ...

![image](https://github.com/in2code-de/powermail/assets/13206455/1c03e8a2-8985-4e92-ba66-2bf1cad57d54)

When you click on "more", the answers for the different plugin are displayed.

## Expected behaviour

* possibly, instead of or in addition to the uid of the form, the uid of the plugin are used to create a reference from the answer to the plugin/ form
* or, make it possible to only use a form in one plugin (but this is tedious it is then necessary to create a form and probably duplicate lots of them)

## versions

* TYPO3 11.5.33
* powermail: 10.7.4

**Comment by mschwemer** [comment]
Yes, it is intended that a form can be used in several plugins. 

I think, I remember situations, where I saw the same behaviour. It should be definitly fixed. Maybe not in V12, but in V13 of powemail.
